patroni/postgresql.py

```diff
@@ -375,7 +375,10 @@ def reload_config(self, config):
             self._replace_pg_hba()
 
         if conf_changed or hba_changed:
+            logger.info('Configuration items changed, reloading configuration.')
             self.reload()
+        else:
+            logger.info('No configuration items changed, nothing to reload.')
```

Comment:
I would suggest changing the message to `No postgresql configuration items changed, nothing to reload.` to make it look different from `config.py`